streamer/iceberg_streamer.py

```diff
@@ -0,0 +1,85 @@
+import os
+import json
+from pyiceberg.catalog import load_catalog
+from pyarrow.flight import FlightServerBase, RecordBatchStream
+import logging
+
+logger = logging.getLogger(__name__)
+port = 8085
```

Comment:
q: should we make this configurable via helm?

Reply:
tbh what matters is that the service port is configurable not so much the container app's port, unless we eventually decide to run something else on it (pretty unlikely)